cl/sentinel/service/start.go

```diff
@@ -36,6 +39,14 @@ func generateSubnetsTopics(template string, maxIds int) []sentinel.GossipTopic {
 	return topics
 }
 
+func getExpirationForTopic(topic string) time.Time {
+	if strings.Contains(topic, "beacon_attestation") || strings.Contains(topic, "sync_committee") {
+		return time.Unix(0, 0)
```

Comment:
means do not listen these two topics by default?

Reply:
hmm, so your idea is: 
by default we don't listen to these 2 topics (but can relay / publish), just call `OverwriteSubscriptionExpiry()` to setup appropriate expiration once we need them (eg: beacon_committee_subscription api is invoked )

Reply:
yes